sci-libs/Shapely/metadata.xml

```diff
@@ -1,8 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-  <!-- maintainer-needed -->
+  <maintainer type="person">
+    <email>expeditioneer@gentoo.org</email>
+    <name>Dennis Lamm</name>
+  </maintainer>
```

Comment:
Would you like to add `proxy-maint` project here as well, so we can proxy-commit for you?

Reply:
Done